pkg/kubelet/dockershim/network/hostport/hostport_manager_test.go

```diff
@@ -387,7 +387,7 @@ func TestGetHostportChain(t *testing.T) {
 
 func TestHostportManagerIPv6(t *testing.T) {
 	iptables := NewFakeIPTables()
-	iptables.ipv6 = true
+	iptables.protocol = utiliptables.ProtocolIPv4
```

Comment:
`ProtocolIPv6`
(that's the other breakage in pull-kubernetes-bazel-test)

Reply:
ACk